devel/run-e2e.sh

```diff
@@ -20,12 +20,15 @@ set -o pipefail
 
 # This script will run the end-to-end test suite against an already configured
 # kind cluster.
-# If a cluster does not already exist, create one with 'cluster/create.sh'.
+# If a cluster does not already exist, create one with 'cluster/create-kind.sh'.
```

Comment:
+1 to moving these files to be named as they are, but would it make sense to keep a `create.sh` script too to a) not break existing dev setups and b) auto select what we test against? i.e. if the user sets `IS_OPENSHIFT=true`, the `create-openshift.sh` script is called instead?

Reply:
+1